generatorreceiver/internal/generator/trace_generator_test.go

```diff
@@ -0,0 +1,116 @@
+package generator
+
+import (
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/lightstep/telemetry-generator/generatorreceiver/internal/topology"
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+)
+
+// scafolding for tests
+var topoTestFrontend = topology.ServiceTier{
```

Comment:
Can we also add a couple tag sets to the topology? And then check that the spans get tagged appropriately.

Reply:
Added :)